src/constants/breadcrumbLabels.js

```diff
@@ -0,0 +1,76 @@
+module.exports = {
+  breadcrumbLabels: [
```

Comment:
Could this be generated from `src/constants/Navigation.js`? These files look mighty similar now.

Reply:
Sounds good, but maybe let's keep this as is for now? Once we get this PR on prod, we can create another task to rework Navigation.js to generate this array.